decentraland/screens/BuyLandScreen.js

```diff
@@ -110,6 +119,11 @@ export class BuyLandScreen extends React.Component {
         gasParams
       );
 
+      const actionId = await action.getId();
+
+      // Assumes that the asset was already added to the MyLand list (Optimistic UI update)
+      setActionIdForMyAsset(assetId, actionId);
```

Comment:
Is this one of the redux state updates that is persisted using our new middleware?

Reply:
Oh, good catch. It was missing. Done.

Reply:
Thanks! Resolving this